api/src/main/java/org/openmrs/module/openconceptlab/client/OclMapping.java

```diff
@@ -45,6 +45,9 @@
 	@JsonProperty("to_concept_code")
 	private String toConceptCode;
 	
+	@JsonProperty("updated_on")
+	private String updatedOn;

```

Comment:
You could try declaring is as Date and see if it is correctly converted from json.